source/features/mark-merge-commits-in-list.tsx

```diff
@@ -38,21 +38,37 @@ export function getCommitHash(commit: HTMLElement): string {
 	return $('a.markdown-title', commit)!.pathname.split('/').pop()!;
```

Comment:
```suggestion
	return $([
		'a.markdown-title', // Old view style (before November 2023)
		'.markdown-title a',
	], commit)!.pathname.split('/').pop()!;
```

You can use array to comment on each selector